main.js

```diff
@@ -1,3 +1,48 @@
-import { getData } from "./helpers.js";
+import { data, getData } from "./helpers.js";
```

Comment:
Does it work on your sight? because It througs me an error in devtool console.
Please, fix.
![2022-01-17_15-28-03](https://user-images.githubusercontent.com/650352/149718732-15efb2e6-e8ab-4094-bea6-bcbc8d451856.png)